Sources/UIViewExtensions.swift

```diff
@@ -565,9 +565,34 @@ extension UIView {
         self.layer.mask = mask
     }
 
-    /// EZSwiftExtensions
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover
+     */
     public func roundView() {
-        self.layer.cornerRadius = min(self.frame.size.height, self.frame.size.width) / 2
+        self.setCornerRadius(radius: min(self.frame.size.height, self.frame.size.width) / 2)
+        self.layer.borderWidth = 0
+    }
+    
+    /**
+     EZSwiftExtensions - Mask square/rectangle UIView with a circular/capsule cover, with a border of desired color and width around it
+     
+     - Parameter withBorderColor: Color of circuler border
+     - Parameter withBorderWidth: Width of circuler border
+     */
+    public func roundView(withBorderColor color: UIColor, withBorderWidth width: CGFloat) {
```

Comment:
I am thinking if color can be an optional parameter. That way roundView above can become a special case on the roundView with optional color and borderwidth = 0. Besides, a programmer can choose to have no color but some border width.